python/ray/autoscaler/_private/kuberay/node_provider.py

```diff
@@ -446,6 +468,10 @@ def _scale_request_to_patch_payload(
         for node_type, target_replicas in scale_request.desired_num_workers.items():
             group_index = _worker_group_index(raycluster, node_type)
             group_max_replicas = _worker_group_max_replicas(raycluster, group_index)
+            group_num_of_hosts = _worker_group_num_of_hosts(raycluster, group_index)
+            # Account for multi-host worker groups
+            if group_num_of_hosts != 0:
+                target_replicas /= group_num_of_hosts
```

Comment:
Is it possible for this not to be an integer?

Reply:
Removed this logic in [112a448](https://github.com/ray-project/ray/pull/43470/commits/112a448b6bce7362ba96ca3b3d8894e68bee7511).